### PR TITLE
Fix link to parameterized gates docs.

### PR DESCRIPTION
**Pull request by rmlarose**
Previous link was broken due to changes in Cirq docs. 

Another possible spot to link to in the Cirq docs is here: https://cirq.readthedocs.io/en/stable/docs/simulation.html#Parameterized-Values-and-Studies

### CODE REVIEW

**Comment by balopat** [comment]
> ps://cirq.readthedocs.io/en/stable/docs/simulation.html#Parameterized-Values-and-Studies

Parameterized Values and Studies would be better - however that will change too - Studies don't exist anymore. As I'm thinking about this, we probably should create a new page describing parametrized circuits separately.

**Comment by balopat** [comment]
> > ps://cirq.readthedocs.io/en/stable/docs/simulation.html#Parameterized-Values-and-Studies
> 
> Parameterized Values and Studies would be better for now - however that will change too - Studies don't exist anymore. As I'm thinking about this, we probably should create a new page describing parametrized circuits separately.

On second thought: https://cirq.readthedocs.io/en/stable/docs/tutorials/basics.html#Using-parameter-sweeps this is the right link for now.

File: docs/cirq_interface.md [hunk 1]
```diff
@@ -148,6 +148,6 @@ Known gates with no decomposition:
 ### Parametrized circuits
 
 QSimCircuit objects can also contain
-[parameterized gates](https://cirq.readthedocs.io/en/stable/tutorial.html#parameterizing-the-ansatz)
+[parameterized gates](https://cirq.readthedocs.io/en/stable/docs/tutorials/variational_algorithm.html#Parameterizing-the-Ansatz)
```
Comment (balopat) [inline docs/cirq_interface.md:151]: can you change it to https://cirq.readthedocs.io/en/stable/docs/tutorials/basics.html#Using-parameter-sweeps instead?

**Review by 95-martin-orion — APPROVED** [review]
Once this PR is submitted, we will cut a new release and update the devsite to match.

**Pull request merged**